homeassistant/components/reolink/entity.py

```diff
@@ -112,6 +116,9 @@ async def async_added_to_hass(self) -> None:
             self._host.async_register_update_cmd(cmd_key)
         if cmd_id is not None and self._attr_unique_id is not None:
             self.register_callback(self._attr_unique_id, cmd_id)
+        if self._attr_unique_id is not None:
```

Comment:
why would it be None?

Reply:
This was just a mypy thing, I have removed the check and adjusted the typing instead.